### PR TITLE
Casmpet 5364 main

### PR DESCRIPTION
**Pull request by rgross727**
### Summary and Scope
Main - CASMPET-5364: Review and update System Shutdown and Startup Procedure

Incorporate document changes/additions resulting from testing system shutdown/startup procedures. 
Validate updated procedures on wasp, csm V1.0.1, shutdown/startup.

### Issues and Related PRs
CASMPET-5364

### Testing
Validated on wasp with csm V1.0.1 - cabinets not powered off.

Was a fresh Install tested? N - N/A
Was an Upgrade tested? N - N/A
Was a Downgrade tested? N - N/A
If schema changes were part of this change, how were those handled in your upgrade/downgrade testing? N/A

### Risks and Mitigations
Low

### Requires:
N/A

### CODE REVIEW

File: operations/power_management/Prepare_the_System_for_Power_Off.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ An authentication token is required to access the API gateway and to use the `sa
     1. Find the required `templateUuid` value with BOS.
 
        ```bash
-       ncn# cray bos session describe XNAME | grep templateUuid
+       ncn# cray bos session describe boss_session | grep templateUuid
```
Comment (cdelatte-hpe) [inline operations/power_management/Prepare_the_System_for_Power_Off.md:72]: On any ncn run:
```suggestion
     cray bos session describe boss_session | grep templateUuid
```

**Review by cdelatte-hpe — COMMENTED** [review]
just 1 nit, looks good otherwise

File: operations/power_management/Prepare_the_System_for_Power_Off.md (hunk 1 above)
Comment (rgross727) [reply operations/power_management/Prepare_the_System_for_Power_Off.md:72]: The XNAME above is incorrect. "boss_session" should perhaps be bos_session

Comment (rgross727) [reply operations/power_management/Prepare_the_System_for_Power_Off.md:72]: Fixed above corrections.

**Review by kimjensen-hpe — APPROVED** [review]

**Review by cdelatte-hpe — APPROVED** [review]

**Review by gbaker-hpe — APPROVED** [review]

**Pull request merged**